guardian/guardian-management
# Fix snyk.yaml

- Removes schedule - snyk does daily checking so not needed on our side
- Fixed comments - they are now accurate
- Remove debug - unnecessarily verbose output, can cause intermittent failures

## How to test

Run the snyk action on this branch and verify it passes

<img width="540" alt="Screenshot 2023-04-11 at 09 46 11" src="https://user-images.githubusercontent.com/67543397/231106436-4fbcd2fd-e3ec-4418-8d2a-ac26924452c6.png">
